openedx/core/djangoapps/user_api/api.py

```diff
@@ -415,14 +415,23 @@ def _add_password_field(self, form_desc, required=True):
         # meant to hold the user's password.
         password_label = _(u"Password")
 
+        restrictions = {
+            "min_length": password_min_length(),
+            "max_length": password_max_length(),
+        }
+
+        complexities = getattr(settings, 'PASSWORD_COMPLEXITY', {})
+        if not settings.FEATURES.get('ENFORCE_PASSWORD_POLICY', False):
+            complexities = {}
```

Comment:
Nit: You could swap these conditions and only do the `PASSWORD_COMPLEXITY` lookup if the feature flag is enabled. Doing so also makes the if logic a little clearer since it wouldn't need to include `not`.

Reply:
Yeah you're probably right. Will switch.